benchmarks/bench_cold_start.py

```
@@ -0,0 +1,156 @@
+
+    for scale in scales:
+        docs = all_docs[:scale]
+        total_chars = sum(len(d['content']) for d in docs)
+        print(f'\n--- {scale} docs ({total_chars:,} chars) ---')
+        print(f'{"Framework":<15} {"Total (ms)":>12} {"Chunk (ms)":>12} {"Idx+Search":>12} {"Chunks":>8} {"Mem MB":>8}')
+        print('-' * 72)
+
+        for name, func in CHUNKERS.items():
+            try:
+                r = benchmark_cold_start(name, func, docs, query)
+            except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)